AddressApp.ConsoleApp/Brokers/Loggings/LoggingBroker.cs

```diff
@@ -0,0 +1,21 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace AddressApp.ConsoleApp.Brokers.Loggings
+{
+    internal class LoggingBroker : ILoggingBroker
+    {
+        public ILogger logger { get; set; }
```

Comment:
Run code cleanup and correct the formats

Reply:
Extra Lines removed. Please Check